src/Templates/src/templates/maui-blazor/App.xaml

```diff
@@ -6,20 +6,11 @@
     <Application.Resources>
         <ResourceDictionary>
 
-            <Color x:Key="PageBackgroundColor">#512bdf</Color>
-            <Color x:Key="PrimaryTextColor">White</Color>
-
-            <Style TargetType="Label">
-                <Setter Property="TextColor" Value="{DynamicResource PrimaryTextColor}" />
-                <Setter Property="FontFamily" Value="OpenSansRegular" />
-            </Style>
-
-            <Style TargetType="Button">
-                <Setter Property="TextColor" Value="{DynamicResource PrimaryTextColor}" />
-                <Setter Property="FontFamily" Value="OpenSansRegular" />
-                <Setter Property="BackgroundColor" Value="#2b0b98" />
-                <Setter Property="Padding" Value="14,10" />
-            </Style>
+        <!--
+            For information about styling .NET MAUI pages
+            please refer to the documentation:
+            https://learn.microsoft.com/dotnet/maui/user-interface/styles/xaml
```

Comment:
This should be an FWLink so that we can change it in the future.

Reply:
That makes sense! How do I create those? 😄

Reply:
Oh oops yeah aka.ms is the more modern one. FWLink shows my age 😁 Both are fine though.